pkg/framework/plugins/removepodsviolatingnodeaffinity/node_affinity_test.go

```diff
@@ -233,18 +236,33 @@ func TestRemovePodsViolatingNodeAffinity(t *testing.T) {
 			if tc.strategy.Params != nil {
 				nodeFit = tc.strategy.Params.NodeFit
 			}
+			handle := &frameworkfake.HandleImpl{
+				ClientsetImpl:                 fakeClient,
+				GetPodsAssignedToNodeFuncImpl: getPodsAssignedToNode,
+				PodEvictorImpl:                podEvictor,
+				SharedInformerFactoryImpl:     sharedInformerFactory,
+				EvictorFilterImpl: evictions.NewEvictorFilter(
+					tc.nodes,
+					getPodsAssignedToNode,
+					false,
+					false,
+					false,
+					false,
+					evictions.WithNodeFit(nodeFit),
+				),
+			}
 
-			evictorFilter := evictions.NewEvictorFilter(
-				tc.nodes,
-				getPodsAssignedToNode,
-				false,
-				false,
-				false,
-				false,
-				evictions.WithNodeFit(nodeFit),
+			plugin, err := New(
+				&componentconfig.RemovePodsViolatingNodeAffinityArgs{
+					NodeAffinityType: tc.strategy.Params.NodeAffinityType,
```

Comment:
Would you please remove the `tc.strategy.Params` bits entirely and replace it with `RemovePodsViolatingNodeAffinityArgs`? After all the strategies are migrated the `api.StrategyParameters` type will be deprecated/removed.

Reply:
My bad, rebase + copy mistake